### PR TITLE
Add DoD checklist to GitHub issue templates

### PR DESCRIPTION
**Pull request by to-bar**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -18,3 +18,14 @@ A clear and concise description of any alternative solutions or features you've
 
 **Additional context**
 Add any other context or screenshots about the feature request here.
+
+---
+
+**DoD checklist**
```
Comment (erzetpe) [inline .github/ISSUE_TEMPLATE/feature-request.md:24]: Maybe we can add license update here also for new features? Also we can check if all conversations in PR were resolved.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -23,8 +23,20 @@ A clear and concise description of what you expected to happen.
...
 - OS: [e.g. Ubuntu 18.04.4 LTS, you can use `cat /etc/os-release`]
 
+**epicli version**: [`epicli --version`]
+
 **Additional context**
 Add any other context about the problem here.
+
+---
+
+**DoD checklist**
```
Comment (erzetpe) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: We can check if all conversations in PR were resolved.

**Review by erzetpe — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -23,8 +23,20 @@ A clear and concise description of what you expected to happen.
 If applicable, add config files to help explain your problem.
 
 **Environment**
-- Cloud provider: [AWS | Azure | Any]
+- Cloud provider: [AWS | Azure | All | None]
 - OS: [e.g. Ubuntu 18.04.4 LTS, you can use `cat /etc/os-release`]
```
Comment (plirglo) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Or more generally cat /etc/*release should work on Ubuntu/RH/CentOS, but it's not part of your change

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 1 above)
Comment (atsikham) [reply .github/ISSUE_TEMPLATE/feature-request.md:24]: Isn't it related to `COMPONENTS.md updated / doesn't need to be updated` point?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -23,8 +23,20 @@ A clear and concise description of what you expected to happen.
...
+
+---
+
+**DoD checklist**
+
+* [ ] Changelog updated (if affected version was released)
+* [ ] Automated tests passed (no regression)
+* [ ] Case covered by automated test (if possible)
+* [ ] Idempotency tested
+* [ ] Documentation updated / doesn't need to be updated
```
Comment (atsikham) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I remember that there was an issue caused by Elasticsearch version. WDYT about adding `COMPONENTS.md updated / doesn't need to be updated` here?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (to-bar) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: It seems `cat /etc/os-release` works on all supported distros.

**Review by erzetpe — DISMISSED** [review]

**Review by atsikham — DISMISSED** [review]

**Review by mkyc — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]
Can we have these changes also across the different repos?

**Comment by mkyc** [comment]
> Can we have these changes also across the different repos?

I believe we should.

**Review by mkyc — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**